crypto/ss58.ts

```diff
@@ -1,54 +1,52 @@
 import * as base58 from "../deps/std/encoding/base58.ts"
 import { Blake2b } from "../deps/wat_the_crypto.ts"
 
+export interface EncodeProps {
+  validNetworkPrefixes?: readonly number[]
```

Comment:
Why is this parameter necessary?

Reply:
I think we wanted to let the user specify`validNetworkPrefixes` if any.
@harrysolovay Do you remember?

Reply:
In that case, couldn't the user just validate the network prefix before passing it to encode?

Reply:
makes sense, I've removed the `validNetworkPrefixes`